### DISCUSSION TITLE
Fulfillments endpoint reporting back shipment created, but order still stays in open status

### DISCUSSION THREAD

**Discussion by tjr4206256** [Sellbrite/Sellbrite-API#46] [Q&A]
When marking FedEx SmartPost orders as being shipped it comes back with a 201 shipment created response, but when checking on the orders again it is saying the shipment is still open and tracking is invalid.

**Comment by rengler-godaddy** [comment]
Hello, 

Would you mind posting the request/response(excluding API keys) for one of the orders you’re referencing, and either the email address or company that’s attached to your account?

**Reply by tjr4206256** [reply]
this is what was posted in the request to: https://api.sellbrite.com/v1/shipments

    "shipment": {
    "sb_order_seq": "17619",
    "carrier_name": "FedEx",
    "tracking_number": "9621091390000587933900275251835338",
    "tracking_url": "https://www.fedex.com/fedextrack/?trknbr=9621091390000587933900275251835338",
    "items": [
        {"sku": "PG1W56RH-FBM",
         "quantity": 1
         }],
    }

Comapny: Performance Golf Products Email: support@performancegolf.com

**Comment by rengler-godaddy** [comment]
Thanks for passing that info over. This doesn’t appear to be an API issue as the reason you’re receiving the 201 shipment created response is that the shipment is being accepted by Sellbrite. Once Sellbrite receives the shipment we are passing the fulfillment details to the sales channel, but for some reason, the sales channel is returning an error. I see that you’re currently working with our support team to see why the error is being returned. Once they have an answer as to why we are receiving the error they'll be sure to follow up with you.
